exasol/toolbox/tools/security.py

```diff
@@ -0,0 +1,266 @@
+"""This module contains security related CLI tools and code"""
+import json
+import re
+import subprocess
+import sys
+from dataclasses import (
+    asdict,
+    dataclass,
+)
+from enum import Enum
+from functools import partial
+from inspect import cleandoc
+from typing import (
+    Generator,
+    Iterable,
+    Tuple,
+)
+
+import typer
+
+stdout = print
+stderr = partial(print, file=sys.stderr)
+
+
+# Note: In the long term we may want to adapt the official CVE json schema
+# https://github.com/CVEProject/cve-schema/blob/master/schema/v5.0/CVE_JSON_5.0_schema.json
+@dataclass(frozen=True)
+class Issue:
+    cve: str
+    cwe: str
+    description: str
+    coordinates: str
+    references: tuple
+
+
+def _issues(input) -> Generator[Issue, None, None]:
+    lines = (l for l in input if l.strip() != "")
+    for line in lines:
+        obj = json.loads(line)
```

Comment:
what is if a lines fails to parse, should we catch the exception, print a warning and skip the line?

Reply:
I was thinking about this too ;). As of today the only user is the security issues action.
This means if  the input is flawed something else is broken (not working correctly).
In that case I want it to crash/stop. So if we would catch an error of this I think the call site would be more appropriate. Still I not convinced that we need it right now for the MVP.